src/ddscxx/include/dds/topic/detail/ContentFilteredTopic.hpp

```diff
@@ -34,14 +34,7 @@
 
 // Required for C++11
 #if __cplusplus == 201103L
-namespace std
-{
-template<bool cond, class T = void>
-using enable_if_t = typename std::enable_if<cond, T>::type;
-
-template<class T = void>
-using decay_t = typename std::decay<T>::type;
-}  // namespace std
+#include <org/eclipse/cyclonedds/core/Missing.hpp>
```

Comment:
This is a manner to not have to redefine these missing definitions for older c++ compilers for each file which requires things like std::enable_if, though the name and location of the definition file Missing.hpp is something that may not be the optimal choice.
Should that be changed?